From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rafael Bariccatti <bariccatti@gmail.com>
Date: Sun, 19 Apr 2026 00:00:00 +0000
Subject: [PATCH 1/1] Remove LGE vibrator HAL (depends on removed vibrator@1.0)

android.hardware.vibrator@1.0 was removed in Android 16. The LGE vibrator
HAL depends on it and is not needed for Surface Duo.
---
 .../hardware/vibrator/1.0/Android.bp         | 15 ---------------
 .../hardware/vibrator/1.0/IVibratorEx.hal    |  7 -------
 2 files changed, 22 deletions(-)
 delete mode 100644 vendor/lge/hardware/vibrator/1.0/Android.bp
 delete mode 100644 vendor/lge/hardware/vibrator/1.0/IVibratorEx.hal

diff --git a/vendor/lge/hardware/vibrator/1.0/Android.bp b/vendor/lge/hardware/vibrator/1.0/Android.bp
deleted file mode 100644
index fa69894..0000000
--- a/vendor/lge/hardware/vibrator/1.0/Android.bp
+++ /dev/null
@@ -1,15 +0,0 @@
-// This file is autogenerated by hidl-gen -Landroidbp.
-
-hidl_interface {
-    name: "vendor.lge.hardware.vibrator@1.0",
-    root: "vendor",
-    srcs: [
-        "IVibratorEx.hal",
-    ],
-    interfaces: [
-        "android.hardware.vibrator@1.0",
-        "android.hidl.base@1.0",
-    ],
-    gen_java: true,
-}
-
diff --git a/vendor/lge/hardware/vibrator/1.0/IVibratorEx.hal b/vendor/lge/hardware/vibrator/1.0/IVibratorEx.hal
deleted file mode 100644
index 6a33b87..0000000
--- a/vendor/lge/hardware/vibrator/1.0/IVibratorEx.hal
+++ /dev/null
@@ -1,7 +0,0 @@
-package vendor.lge.hardware.vibrator@1.0;
-
-import android.hardware.vibrator@1.0::IVibrator;
-
-interface IVibratorEx extends IVibrator {
-	playEffectWithStrength(vec<int8_t> effectData, int32_t effectIndex, int32_t strength) generates (int32_t status);
-};
--
2.34.1
